cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/elements/SwitchConversion.java

```diff
@@ -118,6 +118,8 @@ private SwitchKind kind() {
             return SwitchKind.DISCONNECTOR;
         } else if (type.contains("loadbreak")) {
             return SwitchKind.LOAD_BREAK_SWITCH;
+        } else if (type.contains("jumper")) {
+            return SwitchKind.DISCONNECTOR;
```

Comment:
The property bag has for "type" the whole value with the namespace (type -> http://iec.ch/TC57/2013/CIM-schema-cim16#Jumper), but getLocal removes the namespace prefix. In the end, the value we get is exactly the class name. So I guess we could test with "jumper".equals(type) instead.
However, I propose we do it in a separate PR. That way we can change it for all switch types.